feature-libs/organization/administration/components/user-group/services/user-group-item.service.ts

```diff
@@ -46,4 +52,10 @@ export class UserGroupItemService extends ItemService<UserGroup> {
   protected getDetailsRoute(): string {
     return 'userGroupDetails';
   }
+
+  protected launchList() {
```

Comment:
```suggestion
  protected goToList() {
```

Reply:
I used the same name confention as we have in address-item.service